lib/plug/conn.ex

```diff
@@ -590,13 +623,20 @@ defmodule Plug.Conn do
 
   It is recommended for header keys to be in lower-case, to avoid sending
```

Comment:
Not that you changed it, but could we change "lower-case" to "lowercase" to make it the same as its later use in the same paragraph?